packages/server/src/controllers/gateway.controller.ts

```
@@ -14,14 +14,14 @@ import {
 import { Server, Socket } from 'socket.io';
 import {
   logger,
-  metrics,
   WebsocketAgentAddRequestDTO,
   WebsocketAgentDeleteRequestDTO,
   WebsocketAgentRequestDTO,
   WebsocketGetAgentsConfigRequestDTO,
   WebsocketGetMessagesRequestDTO,
   WebsocketSupervisorRequestDTO,
 } from '@snakagent/core';
+import { metrics } from '@snakagent/metrics';
```

ESLint

[error] 'metrics' is defined but never used.

(@typescript-eslint/no-unused-vars)